test/runtests.jl

```diff
@@ -2,78 +2,142 @@ using Test
 using SideKicks
 using Random
 using Turing
+#using LinearAlgebra # RTW should this be here?
+#using Symbolics
+#using IfElse
+#include("./SymbolicFuncs.jl")
+#using .SymbolicFuncs: create_symbolic_functions_list
 
 #Test Kepler's laws using earth sun system
-year_in_days = SideKicks.kepler_P_from_a(au,1,m_earth/m_sun)
+year_in_days = SideKicks.kepler_P_from_a(;m_1=1, m_2=m_earth/m_sun, a=au)
 @test abs(year_in_days-365.25) < 0.01
-au_from_kepler = SideKicks.kepler_a_from_P(year_in_days,1,m_earth/m_sun)
+au_from_kepler = SideKicks.kepler_a_from_P(;m_1=1, m_2=m_earth/m_sun, P=year_in_days)
 @test isapprox(au, au_from_kepler)
+earth_vel = SideKicks.average_orbital_velocity(;m_1=1, m_2=m_earth/m_sun, a=au)
+@test abs(earth_vel-29.7846) < 0.01
+
+#check K1 and K2 using sun-earth system
+RV_e = SideKicks.RV_semiamplitude_K1(;m_1=m_earth/m_sun, m_2=1, P=year_in_days, e=0, i=π/2)
+RV_s = SideKicks.RV_semiamplitude_K1(;m_1=1, m_2=m_earth/m_sun, P=year_in_days, e=0, i=π/2)
+@test abs(RV_e-29.7846) < 0.001 #just test agains value that was computed in a trusted version
+@test isapprox(RV_e/RV_s, m_sun/m_earth) #test that RV ratio follows mass ratio
 
 #Test that a Blaauw kick removing a bit more than half of the mass of a system unbinds the orbit, and one just below does not
-orbit_bound = SideKicks.post_kick_parameters_P(10,0.4999,0,0,0)
-orbit_unbound = SideKicks.post_kick_parameters_P(10,0.5001,0,0,0)
-@test isnan(orbit_bound[1]) && !isnan(orbit_unbound[1])
-
-#Repeat using post_kick_parameters_a
-orbit_bound = SideKicks.post_kick_parameters_a(au,0.4999,0,0,0)
-orbit_unbound = SideKicks.post_kick_parameters_a(au,0.5001,0,0,0)
-@test isnan(orbit_bound[1]) && !isnan(orbit_unbound[1])
-
-#Verify consistency between both kick functions
-m1_i = 10
-m2_i = 15
-frac = 0.8
-m2_f = m2_i*frac
-mtilde = (m1_i+m2_f)/(m1_i+m2_i)
-vkick_div_vrel = 0.1
+orbit_bound =   SideKicks.post_supernova_circular_orbit_a(;m_1i=10, m_2i=20, a_i=au, m_2f=5.0001) 
```

Comment:
I think you don't need the `;` in here, once you start giving keyword arguments julia understands it.